README.md

```diff
@@ -135,6 +135,23 @@ Read detailed instructions at:
 * https://wiki.easyrpg.org/development/compiling/player/windows/visual-studio
 
 
+### Building a libretro core:
+
+Building for libretro is based on the CMake method.
+
+Additional commands required before building:
```

Comment:
Generally, I would use a subrepo nowadays. https://github.com/ingydotnet/git-subrepo#benefits
(No additional maintenance burden, users can use the repository checkout as is.)

However, as libretro is not the normal build and the repository files would mostly be useless, a submodule is the better approach of course.